Sources/Components/Components/Onboarding/WKOnboardingView.swift

```diff
@@ -0,0 +1,70 @@
+import SwiftUI
+
+public struct WKOnboardingView: View {
+
+    // MARK: - Properties
+
+    var viewModel: WKOnboardingViewModel
+    var mainButtonAction: (() -> Void)?
+    var secondaryButtonAction: (() -> Void)?
+
+    @ObservedObject var appEnvironment = WKAppEnvironment.current
+
+    @Environment (\.horizontalSizeClass) private var horizontalSizeClass
+
+    var sizeClassPadding: CGFloat {
+        horizontalSizeClass == .regular ? 64 : 32
+    }
+
+    // MARK: - Lifecycle
+
+    public var body: some View {
+        ZStack {
+            Color(appEnvironment.theme.paperBackground)
+                .ignoresSafeArea()
+            ScrollView(showsIndicators: true) {
+                VStack {
+                    Text(viewModel.title)
+                        .font(WKFont.for(.title).bold())
+                        .padding([.bottom, .top], 44)
+                        .multilineTextAlignment(.center)
+                    ForEach (1...viewModel.cells.count, id:\.self) { cell in
+                        VStack {
+                            WKOnboardingCell(viewModel: viewModel.cells[cell - 1])
+                                .padding([.bottom, .trailing], 20)
+                        }
+                    }
+                    Spacer()
+                    VStack {
+                        Button(action: {
```

Comment:
Maybe optional, but it might be a good time to treat these big buttons as separate components that can be reused elsewhere. You could move them and their modifiers into their own SwiftUI View under Components > Shared > Buttons directory. If you do this, I'd say avoid "Onboarding" in the name, in case we can reuse them in non-onboarding places in the future.